lighthouse-core/report/html/report-styles.css

```diff
@@ -82,6 +82,33 @@
   --chevron-size: 12px;
   --circle-size: calc(3 * var(--header-font-size));
 
+  --color-black-100: #F5F5F5;
```

Comment:
doesn't seem like we use most of these, just copied and prepping for future?

Reply:
yeah. there's gonna be a last step for cleaning up stuff. I found myself doing a lot of back and forth copying and pasting these values as I came across them.

Reply:
> there's gonna be a last step for cleaning up stuff.

in general that sentence scares me a bit with CSS 😆 

I wish in addition to unused CSS tools there were "useless CSS" tools. i.e. this property you have set isn't actually doing anything, but I dream :)

Reply:
> I wish in addition to unused CSS tools there were "useless CSS" tools. i.e. this property you have set isn't actually doing anything, but I dream :)

i can dig it

curate many different reports, render in Chrome w/ a couple different viewports, attach a DT session to each and for each node, record which rules(*) are active. The rules that were never marked active are always useless and can be remove. bonus points if you toggle spacing rules (margin, padding, max-*/min-* etc) to see if the node changes spacing at all. if not, delete that rule.

* i assume this must be possible since the DT inspector shows rules as grayed out when they are overridden by a more specific rule.

Reply:
That's a nice approach! I'm also thinking the rules that are technically active they just have no effect on the final display (maybe that's just your bonus points 😃)

Reply:
take it a step further and make screenshots after each change :P

I want this tool now

Reply:
bump on 


    /* TODO(cjamcl) clean up unused variables. */